templates/go/base/params.twig

```diff
@@ -10,7 +10,11 @@
 	params["{{ parameter.name }}"] = {{ parameter.name | caseUcfirst }}
 {% else %}
 	if options.enabledSetters["{{ parameter.name | caseUcfirst}}"] {
+	{%~ if method.name | caseLower == "createexecution" and parameter.name == "body" %}
+		params["{{ parameter.name }}"] = string(options.{{ parameter.name | caseUcfirst }}.Data)
```

Comment:
this again feels too specific to createexecution... 

why dont we copy over the stuff from php ?